src/AffordableMobiles/GServerlessSupportLaravel/Foundation/ApplicationBuilder.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+declare(strict_types=1);
+
+namespace AffordableMobiles\GServerlessSupportLaravel\Foundation;
```

Comment:
Just noticed the location on this, do you mind if we try to match the Laravel folder structure before we merge please? So this in `Foundation\Configuration` ideally please.